environment.ts

```diff
@@ -0,0 +1,8 @@
+// The list of file replacements can be found in `angular.json`.
+
+export const environment = {
+    production: false,
+    facebookphotosToken: 'EAAE3R0omZCJUBAHvuLABjVTnHiHtVQrtktFRcNB8cjscTC3hZBsx2k9urAEcbMWJeF7HVb48oDaS99glfxxSRSjUpyZCtAWyGLs3ZAqTEyYs6PKVwVEcFRWY4Fi105kgURZAMRPEltg35adsvnQnhpdKYQ0yhaIqtN6ZALJDxNl9Bn9zZCXL6Kx7p3aKm6JlY8repc4U3QHCwZDZD',
```

Comment:
This is a good practice but just let you know that exposing API token can be dangerous for privacy or data usage.
But in this case, it's a testing account it should be fine.
BTW for other participants wanna test for this API my suggestion will be adding your own API key in the local machine.